File: Application.Server/Models/DTOs/GetBooking/WorkspaceDto.cs
using Application.Server.Models.CoworkingDatabase;

namespace Application.Server.Models.DTOs.GetBooking
{
    public class WorkspaceDto
    {
        public int Id { get; set; }
        public int Capacity { get; set; }
        public List<Amenity> Amenities { get; set; } = null!;
        public WorkspaceType WorkspaceType { get; set; }
    }
}

File: application.client/src/DTOs/GetBooking/WorkspaceDto.ts
import { Amenity } from "../../models/Amenity"
import { WorkspaceType } from "../../models/WorkspaceType"
import { BookingDto } from "./BookingDto";

export interface WorkspaceDto
{
  id: number,
  capacity: number,
  amenities: Amenity[]
  workspaceType: WorkspaceType
}
